sonaranalyzer-dotnet/rspec/cs/S2077_c#.html

```diff
@@ -0,0 +1,81 @@
+<p>Executing SQL queries is security-sensitive. It has led in the past to the following vulnerabilities:</p>
+<ul>
+  <li> <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-9019">CVE-2018-9019</a> </li>
+  <li> <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-7318">CVE-2018-7318</a> </li>
+  <li> <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-5611">CVE-2017-5611</a> </li>
+</ul>
+<p><a href="https://www.owasp.org/index.php/SQL_Injection">SQL injection</a> is still one of the top 10 security vulnerabilities. Applications that
+execute SQL commands should neutralize any externally-provided values used in those commands. Failure to do so could allow an attacker to include
```

Comment:
Question for @nicolas-harraudeau-sonarsource: wouldn't we normally use "sanitize" in this context?

Reply:
It's true that "sanitize" will create less confusion as we are using this word in the rest of the RSPEC. I'll change that.
As a side note, CWE and OWASP texts use both "neutralization" and "sanitization" in this context. Example: https://cwe.mitre.org/data/definitions/89.html

Reply:
@val Change in the RSPEC done